src/components/map/map-control.js

```diff
@@ -240,16 +242,22 @@ const Toggle3dButton = React.memo(({dragRotate, onTogglePerspective}) => (
 
 Toggle3dButton.displayName = 'Toggle3dButton';
 
-const StyledToolBar = styled(Toolbar)`
+const StyledToolbar = styled(VerticalToolbar)`
   position: absolute;
   right: 32px;
 `;
 
-const MapDrawPanel = React.memo(({editor, isActive, onToggleMenuPanel, onSetEditorMode}) => {
+const MapDrawPanel = React.memo(({
```

Comment:
In split map, is it possible to only show `MapDrawPanel` in the split map clicked?  Currently, it is shown in both, which is a little weird.
e,g, in `mapControls.mapDraw` we can save the `splitMap` index [true, false], instead of `true | false`. However, this means in `ui-state` reducer has to listen to `TOGGLE_SPLIT_MAP` and update mapDraw

<img width="966" alt="Screen Shot 2020-01-07 at 9 18 34 PM" src="https://user-images.githubusercontent.com/3605556/71953746-fc3e8580-3197-11ea-95ad-72cc62849d15.png">

Reply:
We may have a similar problem with the legend being displayed in both maps at the sam time
hmm do you think a lot of people will be using the editor mode in split map view?
We could disable the editor when in split mode

Reply:
displaying legend and map control panel in 2 places is fine actually. It allows people to compare whats different in 2 split map. But allow drawying in split maps is useful. you can imagine people want to draw in one map, and see how the other map is filtered. 

For example. showing begin trip in one map, and dropoff in another. Drawing in begin trip to see how dropoff map changes...

Reply:
ok, that makes sense